### PR TITLE
[RFC] Bump up packages

### PR DESCRIPTION
**Pull request by sskartheekadivi**
The following packages are updated in this commit
- mesa
- ti-img-rogue-driver
- ti-img-rogue-umlibs
- ti-linux-firmware
- ti-linux-kernel
- ti-linux-kernel-rt

### CODE REVIEW

**Comment by sskartheekadivi** [comment]
I'm still building and testing the packages. So do not merge yet.

File: mesa/suite/bookworm/debian/patches/0001-PVR-changes.patch [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-@@ -116,7 +116,12 @@ foreach d : [[with_gallium_kmsro, [
-              [with_gallium_zink, 'zink_dri.so'],
-              [with_gallium_d3d12, 'd3d12_dri.so'],
-              [with_gallium_asahi, 'asahi_dri.so'],
--             [with_gallium_pvr, 'pvr_dri.so'],
-+             [with_gallium_pvr, [
-+               'pvr_dri.so',
-+               'tilcdc_dri.so',
-+               'omapdrm_dri.so',
-+               'tidss_dri.so',
```
Comment (glneo) [inline mesa/suite/bookworm/debian/patches/0001-PVR-changes.patch:24]: I believe we will still need this, even if we do not plan to support the SGX boards.

**Comment by glneo** [comment]
Might be good to split  these into commits by package. The `ti-linux-firmware` package should be fine and can go in independent of the other changes.

File: mesa/suite/bookworm/debian/patches/0001-PVR-changes.patch (hunk 1 above)
Comment (sskartheekadivi) [reply mesa/suite/bookworm/debian/patches/0001-PVR-changes.patch:24]: yup. I've added them locally. Will push once after my build is done.

**Comment by sskartheekadivi** [comment]
> Might be good to split these into commits by package. The `ti-linux-firmware` package should be fine and can go in independent of the other changes.

Yes will be splitting it up once my builds are done and tested.

**Comment by sskartheekadivi** [comment]
Created a commit for updating to 09.02.00.006. So closing this one.

**Pull request closed without merging**